airflow/providers/google/suite/hooks/drive.py

```diff
@@ -128,31 +128,45 @@ def get_media_request(self, file_id: str) -> HttpRequest:
         request = service.files().get_media(fileId=file_id)
         return request
 
-    def exists(self, folder_id: str, file_name: str, drive_id: Optional[str] = None):
+    def exists(
+        self, folder_id: str, file_name: str, include_trashed: bool = True, drive_id: Optional[str] = None
```

Comment:
`include_trashed` is also optional param is it not?

Reply:
It is :) this is why we set it with a default value..
Setting is with `Optional[]` and with default value is redundant, as I'd we don't pass it, it's passed with it's default value..

Am I missing something?

Reply:
Just to be sure, I checked the official documentation of typing on `Optional[]`
https://docs.python.org/3/library/typing.html#typing.Optional
They also state the following
```
An optional argument with a default does not require the Optional qualifier ....
```